public/ng-views/maps.html

```diff
@@ -512,23 +577,29 @@ <h5 ng-show="variants[tabnav].sel">
             {{ scenario[tabnav].offpeak.min | avgWaitTime }}
           </td>
           <td>
-            <span class="badge pull-right" ng-click="createVar = !createVar">
+            <!-- <span class="badge pull-right" ng-click="createVar = !createVar">
               <span ng-show="createVar">
                 <i class="fa fa-times"></i> Close
               </span>
               <span ng-show="!createVar">
                 <i class="fa fa-bookmark-o"></i> Modify
               </span>
-            </span>
+            </span> -->
           </td>
         </tr>
       </tbody>
     </table>
 
   </div>
+</div>
+<!--End of ROUTE TABLE MODULE-->
+
+<!--ROUTE EDITOR MODULE--> 
 
-  <div class="col-xs-12 body" ng-show="createVar">
-    <div class="row" style="border-bottom-width:0px;border-bottom-style:solid">
+<div class="basecol_serviceEditor col-md-3">
+  <h5> AVG WAIT TIME </h5>
```

Comment:
Why is this all caps? This is a graphic design question but there is 0 consistency with the headers, which makes legibility quite difficult - I am having trouble finding where things are and I should be familiar with it! :)